src/dotnet/Program.cs

```diff
@@ -81,6 +81,7 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient = null
             var lastArg = 0;
             var cliFallbackFolderPathCalculator = new CliFallbackFolderPathCalculator();
             using (INuGetCacheSentinel nugetCacheSentinel = new NuGetCacheSentinel(cliFallbackFolderPathCalculator))
+            using (IFirstTimeUseNoticeSentinel firstTimeUseNoticeSentinel = new FirstTimeUseNoticeSentinel(cliFallbackFolderPathCalculator))
```

Comment:
@livarcocc @wli3 is `cliFallbackFolderPathCalculator` a per user thing or is it changing to be some system wide path?  In the case of it being system wide, I think we should be writing the telemetry sentinal message to some user specific folder, do you agree?